tests/integration/multicluster/routing.go

```diff
@@ -0,0 +1,116 @@
+// Copyright Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package multicluster
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	"istio.io/istio/pkg/test/framework"
+	"istio.io/istio/pkg/test/framework/components/echo"
+	"istio.io/istio/pkg/test/framework/components/echo/echoboot"
+	"istio.io/istio/pkg/test/framework/components/namespace"
+	"istio.io/istio/pkg/test/framework/components/pilot"
+	"istio.io/istio/pkg/test/framework/features"
+	"istio.io/istio/pkg/test/framework/label"
+	"istio.io/istio/pkg/test/framework/resource"
+	"istio.io/istio/pkg/test/util/retry"
+)
+
+// RoutingTest verifies that traffic resources created in a cluster are reflected in the
+// configs a sidecar receives in multicluster scenarios.
+func RoutingTest(t *testing.T, ns namespace.Instance, pilots []pilot.Instance, feature features.Feature) {
```

Comment:
do we need to recreate TestTrafficRouting or can we reuse it?

Reply:
@howardjohn : @stevenctl was doing some work on migrating tests to support multicluster - I am not sure where that work stands but TestTrafficRouting is not set up for multicluster as it is now.  I agree it should eventually use the same test, but doing so seems outside the scope of this PR.  For tracking purposes, I can create an issue and assign it to myself if you want

Reply:
https://github.com/istio/istio/pull/23969/ will contain the topology we run the existing ITs in; maybe centralistiod is all we need if we plan on that replacing the existing shared control-plane model. 

Hopefully the MC tests will land soon and we could just call a func from the pilot/ package to trigger the test which will just use all clusters the test framework knows about.